aspnetcore/host-and-deploy/iis/hosting-bundle.md

```diff
@@ -38,33 +45,30 @@ To obtain an earlier version of the installer:
 > [!WARNING]
 > Some installers contain release versions that have reached their end of life (EOL) and are no longer supported by Microsoft. For more information, see the [support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
 
-### Install the Hosting Bundle
+## Options
 
-1. Run the installer on the server. The following parameters are available when running the installer from an administrator command shell:
+1.  The following parameters are available when running the installer from an administrator command shell:
 
    * `OPT_NO_ANCM=1`: Skip installing the ASP.NET Core Module.
    * `OPT_NO_RUNTIME=1`: Skip installing the .NET Core runtime. Used when the server only hosts [self-contained deployments (SCD)](/dotnet/core/deploying/#self-contained-deployments-scd).
    * `OPT_NO_SHAREDFX=1`: Skip installing the ASP.NET Shared Framework (ASP.NET runtime). Used when the server only hosts [self-contained deployments (SCD)](/dotnet/core/deploying/#self-contained-deployments-scd).
    * `OPT_NO_X86=1`: Skip installing x86 runtimes. Use this parameter when you know that you won't be hosting 32-bit apps. If there's any chance that you will host both 32-bit and 64-bit apps in the future, don't use this parameter and install both runtimes.
    * `OPT_NO_SHARED_CONFIG_CHECK=1`: Disable the check for using an IIS Shared Configuration when the shared configuration (`applicationHost.config`) is on the same machine as the IIS installation. *Only available for ASP.NET Core 2.2 or later Hosting Bundler installers.* For more information, see <xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration>.
-1. Restart the system or execute the following commands in a command shell:
 
-   ```console
-   net stop was /y
-   net start w3svc
-   ```
-   Restarting IIS picks up a change to the system PATH, which is an environment variable, made by the installer.
+> [!NOTE]
+> For information on IIS Shared Configuration, see [ASP.NET Core Module with IIS Shared Configuration](xref:host-and-deploy/aspnet-core-module#aspnet-core-module-with-an-iis-shared-configuration).
+
+## Restarting IIS
+
+On install, the hosting bundle may require IIS to restart. For example, this can be due to `dotnet` not being on the PATH for already running IIS worker processes.
```

Comment:
🤔 I think we were going with the proper noun everywhere back in the day. We *might* be able to avoid the state verb here, too. See if something like this works (and you can edit this comment directly, change the suggestion, then add it to the batch ... forgive me, you probably already know that 😉) ...

```suggestion
On install, the Hosting Bundle may require a manual IIS restart. For example, the `dotnet` CLI tooling (command) might not exist on the PATH for running IIS worker processes.
```